### ISSUE TITLE
Does GoAccess GeoIP  supports City coordinates  ?

### ISSUE THREAD

**Issue by dawapaljor** [allinurl/goaccess#1498]
I have Maxmind mmdb (GeoIP2-City_20190108.tar.gz). So when I configured in Goaccess. Geolocation shows only CONTINENT > COUNTRY. City is not showing.  
in my configuration file(goaccess.conf)  **geoip-database**  is set to GeoIP2-City_20190108.mmdb
 Does anyone have any Idea please help me to figure it out.  
`i am not using GeoIpCityLite

**Comment by allinurl** [comment]
City DB is used for displaying city stats per IP. Please take a look at the `Visitor Hostnames and IPs` panel. #206 should address this.

**Comment by dawapaljor** [comment]
@allinurl  thanks for the response.  I got an ideas from what you share thanks. So is there a way to display the Geo coordinates like **Region>City**  after the **continent>Country**  Identifying the Geo cordinates of city is important for us.. thanks.

**Comment by dawapaljor** [comment]
@allinurl thanks for the response. I got an ideas from what you share thanks. So is there a way to display the Geo coordinates like Region>City after the continent>Country Identifying the Geo cordinates of city is important for us.. thanks.

**Comment by allinurl** [comment]
Not at the moment, but are you looking to have a new panel for this or within the existing continent/country panel?

**Comment by dawapaljor** [comment]
@allinurl Yes by drilling down **Continent>country>Region>City**  In whatever possible ways that would be helpful from the UI perspective.

**Comment by allinurl** [comment]
Thanks for the feedback, will look into this!

**Comment by favoyang** [comment]
+1 would like to have city level stats in `GEO LOCATION` panel.

**Comment by canuckdev** [comment]
+1 also would like to have city level stats in GEO LOCATION panel.

**Comment by allinurl** [comment]
This has been implemented in the [latest version](https://goaccess.io/). v1.10.1. e.g., 

<img width="712" height="530" alt="Image" src="https://github.com/user-attachments/assets/edfe57b6-5684-4f00-8125-74bb42ae47d4" />
